src/components/AppFrame/AppFrame.js

```diff
@@ -74,6 +94,12 @@ const AppFrame = ({ initOpen, applicationId, modules, activeModules, children, n
 	return (
 		<Base preferencesOpen={prefViewState.show}>
 			<ConnectedToastList />
+			<ActionModal
+				title={formatMessage(sharedMessages.error)}
+				message={formatMessage(sharedMessages.needToRefresh)}
```

Comment:
While you are in orc-shared I would like for you to add to ActionModal message a default styling. Currently the text is too big.  it should be using 

fontSize: theme.typography.fontsize,
fontFamily: theme.typography.fontFamily,